modules/ROOT/pages/backup-and-disaster-recovery.adoc

```diff
@@ -94,6 +97,57 @@ Outputs logs to a file.
 
 . Manually restore the NFS files if you are not reusing the original NFS.
 
+== Anypoint Monitoring and Visualizer
+
+If Anypoint Monitoring and Visualizer are enabled, follow these procedures to back up and restore them.
+
+Anypoint Monitoring and Visualizer backup and restore are outside the normal Gravity backup and restore mechanism, and require additional procedures.
+
+These components store metrics and other information, but they do not store platform configuration.
```

Comment:
These components store metrics and other information, but they do not store platform configuration which is already covered by the main backup and restore procedure

Reply:
Changed to:

Anypoint Monitoring and Anypoint Visualizer backup and restore are outside the normal Gravity backup and restore mechanism, and require additional procedures. These components store metrics and other information, but they do not store platform configuration. Platform configuration information is handled by the system backup and restore procedures.